pytest/tests/loadtest/locust/download_contracts.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+# Downloads the WASM contracts necessary for all workloads and stores them in "res" folder.
+
+cd res
+wget https://raw.githubusercontent.com/NearSocial/social-db/master/res/social_db_release.wasm -O social_db_release.wasm
+wget https://raw.githubusercontent.com/near/nearcore/master/runtime/near-test-contracts/res/fungible_token.wasm -O fungible_token.wasm
+cp ../../../../../runtime/near-test-contracts/res/test_contract_rs.wasm .
```

Comment:
optional nit to save a few kB: Aren't both of these dependencies from nearcore? Would it make sense to use symbolic links for the two?
I guess we would have the problem that `test_contract_rs.wasm` is not built in a fresh checkout but you could instead use `runtime/near-test-contracts/res/backwards_compatible_rs_contract.wasm` which is checked in and should be equivalent aside from the rustc version used to compile it.

but feel free to ignore, checking them in is also fine and probably more robust

Reply:
That works! Done.